client/app/components/TimeAgo.jsx

```diff
@@ -6,6 +6,25 @@ import PropTypes from 'prop-types';
 import { Moment } from '@/components/proptypes';
 import { clientConfig } from '@/services/auth';
 import useForceUpdate from '@/lib/hooks/useForceUpdate';
+import Tooltip from 'antd/lib/tooltip';
+
+moment.updateLocale('en', {
+  relativeTime: {
+    future: '%s',
+    past: '%s',
+    s: 'just now',
```

Comment:
Did we change any of the other strings or only this one?

Reply:
Nay.
I experimented with short-handing ("2 minutes ago" to "2m ago") but there's no straightforward way with `moment` to apply it for widgets only.